src/host/config/KBoxConfigParser.cpp

```diff
@@ -46,7 +46,15 @@ void KBoxConfigParser::defaultConfig(KBoxConfig &config) {
   config.serial2Config.inputMode = SerialModeNMEA;
   config.serial2Config.outputMode = SerialModeDisabled;
 
-  config.imuConfig.enabled = true;
+  config.imuConfig.enabled = true;            // enable internal IMU sensor
+  config.imuConfig.enabledHdg = true;         // true if values taken from internal sensor
```

Comment:
I think this should be `headingEnabled` or `enableHeading`. Otherwise it grammatically sounds incorrect.

Reply:
And maybe `enableCompass`? instead of heading?

Reply:
of course enableXY OMG – english is not my mother language which you probably saw already :-))

Hmm, we have on some of our boats a Gyro-Compass which delivers Heading and Heel (not sure about pitch)
With enableCompass I think it is not clear because it could be meant the whole IMU thing.
The intention is to stop just hdg values from IMU-sensor.